website/docs/reference/integrations/datadog.md

```diff
@@ -28,19 +28,57 @@ You can choose to trigger updates for the following events:
 - feature-stale-off
 - feature-environment-enabled
 - feature-environment-disabled
+- feature-variants-updated
+- feature-potentially-stale-on
 
 > *) Deprecated, and will not be used after transition to environments in Unleash v4.3
 
 #### Parameters {#parameters}
 
 Unleash Datadog integration takes the following parameters.
 
-- **Datadog Events URL** - This property is optional. The default url is https://api.datadoghq.com/api/v1/events. Needs to be changed if you are not not on the US1 [Datadog site](https://docs.datadoghq.com/getting_started/site/). Possible alternatives:
+- **Datadog Events URL** - This property is optional. The default URL is https://api.datadoghq.com/api/v1/events. Needs to be changed if you are not not on the US1 [Datadog site](https://docs.datadoghq.com/getting_started/site/). Possible alternatives:
   - EU: https://app.datadoghq.eu/api/v1/events
   - US1: https://app.datadoghq.com/api/v1/events
   - US3: https://us3.datadoghq.com/api/v1/events
   - US1-FED: https://app.ddog-gov.com/api/v1/events
-- **DD API KEY** - This is a required property.
+
+
+- **Datadog API key** - This is a required property. The API key to use to authenticate with Datadog.
+
+- **Datadog Source Type Name** - This property is optional. Sets `source_type_name` parameter to be included in Datadog events.
+
+- **Extra HTTP Headers** - This property is optional. Used to set the additional headers when Unleash communicates with Datadog.
+
+Example:
+
+```json
+{
+  "ISTIO_USER_KEY": "hunter2",
+  "SOME_OTHER_CUSTOM_HTTP_HEADER": "SOMEVALUE",
+}
+```
+
+- **Body template** - This property will be introduced as `beta` in `5.5` and is expected to enter `GA` in `5.6`. It is optional. Used to override the body template used by Unleash when performing the HTTP POST. You may format you message using a [Mustache template](https://mustache.github.io). You will have the [Unleash event format](/reference/api/legacy/unleash/admin/events) available in the rendering context.
+
+Example:
+
+```mustache
+{
+  "event": "{{event.type}}",
+  "createdBy": "{{event.createdBy}}",
+  "featureToggle": "{{event.data.name}}",
+  "timestamp": "{{event.data.createdAt}}"
+}
+```
+
+If you don't specify anything Unleash will send a formatted markdown body.
+
+Example:
+
+```markdown
+username created feature toggle (featurename)[http://your.url/projects/projectname/features/featurename] in project *projectname*
+```
```

Comment:
I think the availability not can go down here to call it out. We can also say that it's currently under development and expected to go out in the next few releases, but I think this is fine.

```suggestion
```

:::info Body template availability

The body template property will be introduced as a beta feature in Unleash 5.5 and is expected to be made generally available in Unleash 5.6.

:::
```

Reply:
I'm a bit confused by this comment but I think I understand the core concern of having the availability displayed as usual. Tried addressing it here: https://github.com/Unleash/unleash/pull/4802/commits/fdcc22a0c8bd170c75cc9277e4c772f9503d5745

Reply:
You got it! Looks great 😄